eventstream.go

```diff
@@ -101,7 +101,7 @@ type BrokerConfig struct {
 	BaseWriterConfig *kafka.WriterConfig
 	BaseReaderConfig *kafka.ReaderConfig
 
-	MetricsRegistry *prometheus.Registry // optional registry to report metrics to prometheus (used for kafka stats)
+	MetricsRegistry prometheus.Registerer // optional registry to report metrics to prometheus (used for kafka stats)
```

Comment:
is this considered breaking? since we change the data type?

Reply:
This should not breaking changes because `*prometheus.Registry` implements `prometheus.Registerer`.

Reply:
how about in the monitoring metrics? since it will be used for metrics

Reply:
What is monitoring metrics?

Reply:
I mean for the prometheus metrics, sorry it was the same thing